parlai/tasks/dailydialog/parse.py

```diff
@@ -14,6 +14,7 @@
 import os
```

Comment:
This is reasonably dead code that is left there as reference. We could easily delete the whole script with a comment left about the last commit it lived in.